docs/BrandColors.stories.tsx

```diff
@@ -39,7 +39,12 @@ export const CSS: Story = {
       >
         {/* Mapping through each brand color and rendering a ColorSwatch component for it */}
         {Object.values(cssBrandColors).map(({ color, name }) => (
-          <ColorSwatch key={name} color={color} name={name} />
+          <ColorSwatch
+            key={name}
+            color={color}
+            backgroundColor={name}
```

Comment:
Adds backgrouncColor prop that points to the name which is the css variables

### Before
<img width="757" alt="Screenshot 2024-05-22 at 12 04 42 PM" src="https://github.com/MetaMask/design-tokens/assets/8112138/51cd2bc8-e12f-470d-b746-a3bccfbecc85">

### After
<img width="756" alt="Screenshot 2024-05-22 at 12 06 03 PM" src="https://github.com/MetaMask/design-tokens/assets/8112138/db3c6244-5a7f-4c34-a6e7-8a2db92f286b">